mksfosinitrd.sh

```diff
@@ -84,6 +85,8 @@ cd "$TMP_DIR"
 # Copy local files to be added to initrd. If you add more, add also to TOOL_LIST.
 cp -a "$OLD_DIR"/sbin .
 cp -a "$OLD_DIR"/res .
+mkdir etc
```

Comment:
Here and above, I wonder if we should do mkdir -p if for some reason the directory exists or is more than one dir it would not fail. Just a though, not mandatory to change.